client_src/js/templates/console-log.template.js

```diff
@@ -0,0 +1,24 @@
+export function logToConsole(event) {
```

Comment:
Loggin is suppressed now depending on browser. Needs something... not saying this specifically as is still a W.I.P anyway but to tackle your points. 

1. Yeah depends what information we want to see, I see no issue with this personally if I'm parsing out the values we do want. 

2. The info shouldn't really be there, it will be getting removed and I can remove stack out of warning. 

3. I use these 3 colours not sure on your point here. The event.colour is a requirement for the event.type regardless because it's utilised in the client error pop-ups. 

Example below is brave, without that call. Gather Chrome is the same. Firefox still shows the trace. 

![image](https://user-images.githubusercontent.com/61137988/130344212-6a2642cb-ddf9-4e24-98c1-267697c128de.png)